hw03_rockero241.py

```diff
@@ -0,0 +1,93 @@
+
+#################################################################################
+# Author: Luis Olivas
+# Username: rockero241
+#
+# Assignment: hw03 - Fully Functional Gitty Psychedelic Robot Turtles
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1mrc8WmON440HpzEmCvNJ3uzkPaZT3mTe7Op433ShEAs/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+
+def draw_building(building):
+    """
+    Draws the building using the building turtle.
+    """
+    building.color('gold')
+    building.fillcolor(115, 134, 120)
+    building.begin_fill()
+
+    for i in range(2):
+        building.fd(120)
+        building.right(90)
+        building.fd(240)
+        building.right(90)
+    building.end_fill()
+    # ....
+
+
+def draw_windows(windows):
+    """
+    Draws the building's windows row by row.
+    """
+    # object attributes
+    windows.color(0, 0, 0)
+    windows.fillcolor('gold')
+    windows.hideturtle()
+    # make an entire row of windows
+    def make_window_row(windows):
+        windows.pendown()
+        makeRectangle(windows, 15, 15)
+        windows.penup()
+        windows.fd(52.5)
+        windows.pendown()
+        makeRectangle(windows, 15, 15)
+
+    b = -25 # for row position
+    for i in range(4):
+        a = 35
+        row_position = (a, b)
+        # row position
+        windows.penup()
+        windows.goto(row_position)
+        windows.begin_fill()
+        make_window_row(windows)
```

Comment:
I don't understand why defining the make_window_row function had to be defined inside this function, instead of outside. 

See if you can refactor it and move it out.